docs/rules/object-curly-spacing.md

```diff
@@ -22,28 +22,28 @@ export { foo };
 
 ## Rule Details
 
-This rule aims to maintain consistency around the spacing inside of object literals. It also
-applies to EcmaScript 6 destructured assignment and import/export specifiers.
-
-It either requires or disallows spaces between those braces and the values inside of them.
-Braces that are separated from the adjacent value by a new line are exempt from this rule.
+This rule enforce consistent spacing inside braces of object literals, destructuring assignments, and import/export specifiers.
 
 ## Options
 
-There are two main options for the rule:
+This rule has a string option:
```

Comment:
@scriptdaemon As I said, I don't believe it's clear enough. For example "it has a string option and an object option" in one rule and  "it has a string option OR an object option" in another. That's a really subtle difference for a reader to pick up, especially if English isn't their first language.

My rule of thumb is that if there's a choice between being explicit or not, always choose being explicit. That would change my examples to "This rule has two options, a string option and an object option" and then "This rule has one option, which can be a string option or an object option." Both of which leave little confusion over what is being described.

If there are other rule docs that aren't including the rule option count, then I apologize for missing those as they went through. I think we need to make this the convention going forward.